package.json

```diff
@@ -46,12 +46,12 @@
     "homepage": "https://github.com/textkernel/oneui#readme",
```

Comment:
![Screenshot 2019-08-06 at 10 26 44](https://user-images.githubusercontent.com/19436304/62529894-d6573280-b83f-11e9-9aec-de57ddd4f964.png)
Make sense implement google module that will be responsible for google features(init/get suggestions, etc), because now we can init this component just once, the second init throw the error _mulitiply init_. This error also appears when to want to use _Map_ component with _Location Autocomplete_ component. Google module probably should be a singleton, that allows us to init google just one time.

Reply:
We also thought so in the beginning, but basically that would mean to reimplement LoadScriptNext, so it would be a waste of time. I haven't looked at multiple occurrences on one page, but will check